splice_machine/src/test/java/com/splicemachine/derby/impl/load/UpportIT.java

```diff
@@ -506,8 +506,8 @@ public void testMergeIntoOccupiedPartiallyTableWillUpdate() throws Exception {
 
         ResultSet resultSet = statement.executeQuery();
         //make sure that the bad records list is good
-        //all 5 rows are updated, no extra row inserted
-        validateMergeResults(resultSet,5,0, 0);
+        //no row should be updated, as we only use the pk column for the merge_data_from_file call
+        validateMergeResults(resultSet,0,0, 0);
```

Comment:
do you mean non-PK columns?

Reply:
I meant PK columns. For this test case, all the columns(essentially 'a') passed to merge_data_from_file are PK columns, so after removing the pk columns from the set column list, nothing is left, and no update statement needs to be triggered, and we should see no row updated. That is what I meant. Thanks.

Reply:
got it thanks!